datalad/support/annexrepo.py

```diff
@@ -3650,8 +3676,8 @@ def __call__(self, line):
 
         # so we have a progress indicator, let's deal with it
         action = j['action']
-        download_item = action.get('file') or action.get('key')
-        download_id = (action['command'], action['key'])
+        download_item = action.get('key') or action.get('file')
```

Comment:
"download_item" seems like a weird name at the level of `ProcessAnnexProgressIndicators`.  Regardless of where this is used at the moment, couldn't the item correspond to  an upload or a local copy or any other action on a file/key?

Reply:
indeed!  I guess it i more of an `action_item`, will change now. thanks